src/data/real/ennreal.lean

```diff
@@ -1232,39 +1256,50 @@ begin
     congr, apply nnreal.coe_of_real, exact h }
 end
 
-@[simp] lemma to_real_mul_top (a : ennreal) : ennreal.to_real (a * ⊤) = 0 :=
+@[simp] lemma to_nnreal_mul_top (a : ennreal) : ennreal.to_nnreal (a * ∞) = 0 :=
 begin
   by_cases h : a = 0,
-  { rw [h, zero_mul, zero_to_real] },
-  { rw [mul_top, if_neg h, top_to_real] }
+  { rw [h, zero_mul, zero_to_nnreal] },
+  { rw [mul_top, if_neg h, top_to_nnreal] }
 end
 
+@[simp] lemma to_nnreal_top_mul (a : ennreal) : ennreal.to_nnreal (∞ * a) = 0 :=
+by rw [mul_comm, to_nnreal_mul_top]
+
+@[simp] lemma to_real_mul_top (a : ennreal) : ennreal.to_real (a * ⊤) = 0 :=
+by rw [ennreal.to_real, to_nnreal_mul_top, nnreal.coe_zero]
```

Comment:
Why is one of the lemmas formulated with `∞` and the other with `⊤`? Same thing for all the surrounding lemmas.

Reply:
I did search&replace in the whole file.